ambry-api/src/main/java/com/github/ambry/quota/ThrottlingRecommendation.java

```diff
@@ -22,10 +22,12 @@
  * Class that returns the overall throttling recommendation for all the quotas.
  */
 public class ThrottlingRecommendation {
+  public static final long NO_RETRY_VALUE_FOR_RETRY_AFTER_MS = -1;
```

Comment:
What does this constant mean? It's a bit of ambiguous to me, could you add some comment or rename this constant.

Reply:
done.